catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -506,14 +506,10 @@ protected MetacardType mergeAttributes(MetacardType metacardType) {
             .flatMap(Collection::stream)
             .collect(Collectors.toSet());
 
-    // Guard against empty collection. If the collection is empty,
-    // the MetacardTypeImpl constructor throws an exception.
-    if (!additionalAttributes.isEmpty()) {
-      returnObject =
-          new MetacardTypeImpl(metacardType.getName(), metacardType, additionalAttributes);
-    }
+    // always add extracted attributes
```

Comment:
❓  should this comment clarify why we always add extracted attributes in case those conditions ever change?

Reply:
Updated comment